package.json

```diff
@@ -21,10 +21,13 @@
   "scripts": {
     "ci": "npm-run-all --continue-on-error --parallel test-cover lint",
     "lint": "eslint --max-warnings 0 .",
+    "lint-dockerfile": "docker run --rm -i hadolint/hadolint:latest-alpine < docker/Dockerfile",
```

Comment:
I like this. But I'm not sure about also putting it in the "ci" script because then everyone needs to have Docker installed. Currently, it's just Node that's required. I may prefer to rely on the GitHub action for this. Thoughts?

Reply:
Oh, you didn't add it there. Maybe we agree.

Reply:
I had originally put it the ci script so that it got linted at the same time as the rest. However, then as you said, it introduced the requirement to have Docker installed. So I've kept it separate.